### PR TITLE
Added interactions with CIE log service

### PR DESCRIPTION
**Pull request by vistaarjuneja**

### CODE REVIEW

File: engine/resource/parser_test.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func TestParse(t *testing.T) {
 			Clone: manifest.Clone{
 				Depth: 50,
 			},
-			Deps: []string{"dependency"},
+			Deps:        []string{"dependency"},
```
Comment (tphoney) [inline engine/resource/parser_test.go:51]: can you leave this as is,

File: engine/compiler/util_test.go [hunk 2]
```diff
@@ -93,8 +93,8 @@ func Test_configureSerial(t *testing.T) {
 
 func Test_convertStaticEnv(t *testing.T) {
 	vars := map[string]*manifest.Variable{
-		"username": &manifest.Variable{Value: "octocat"},
-		"password": &manifest.Variable{Secret: "password"},
+		"username": {Value: "octocat"},
```
Comment (tphoney) [inline engine/compiler/util_test.go:96]: can you leave this as is,

File: engine/compiler/clone_test.go [hunk 3]
```diff
@@ -50,7 +50,7 @@ func TestClone(t *testing.T) {
 			RunPolicy:  runtime.RunAlways,
 			WorkingDir: "/drone/src",
```
Comment (tphoney) [inline engine/compiler/clone_test.go:51]: can you leave this as is,

**Review by tphoney — CHANGES_REQUESTED** [review]

**Review by marko-gacesa — CHANGES_REQUESTED** [review]
@vistaarjuneja can you please not touch any code outside of the delegate package. And also move the new code as a subpackage of the delegate package. We'll later rearrange the code.

**Review by marko-gacesa — APPROVED** [review]

**Review by tphoney — APPROVED** [review]

**Pull request merged**